requirements-test.txt

```diff
@@ -3,6 +3,6 @@ docutils==0.12
 pytest==2.9.0
 pytest-cov==2.2.1
 pytest-flake8==0.1
-pymongo==2.8
+pymongo==3.5.1
```

Comment:
Why new pin? If we need pymongo only because we use `ObjectId` trafaret, maybe better add `bson` as dependency instead of `pymongo`?